src/wwwroot/partials/settings/my-plan.html

```diff
@@ -40,8 +41,8 @@
         <spinner class="spinner" ng-class="vm.planStatusInfoLoader ?  'show':'hide'"></spinner>
       </div>
       <div class="flex two flex--wrap">
-        <label class="width--full special uppercase">{{ 'my_plan_renewal_date' | translate }}</label>
-        <p class="renewal-date">{{ vm.resetDate | date :'mediumDate' :'+0' }}</p>
+        <label class="width--full special uppercase">{{ vm.labelDate | translate }}</label>
+        <p class="renewal-date">{{ vm.resetDate | date :'mediumDate' :'+0' }}</p>        
```

Comment:
Seeing the view I understand your decision, it looks really simple. Anyway, I still prefer a more complex view but having meaningful fields.